<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmDisplay</class>
 <widget class="QWidget" name="FilmDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold 23pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRating">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenre">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Horror, Drama, Comedy, Adult</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLength">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>120 min</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
